tests/e2e-pw/specs/merchant/merchant-subscriptions-renew-action-scheduler.spec.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { test, expect, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import {
+	describeif,
+	getAnonymousShopper,
+	getMerchant,
+	useMerchant,
+} from '../../utils/helpers';
+import * as shopper from '../../utils/shopper';
+import { config } from '../../config/default';
+import {
+	products,
+	shouldRunActionSchedulerTests,
+	shouldRunSubscriptionsTests,
+} from '../../utils/constants';
+import RestAPI from '../../utils/rest-api';
+import {
+	goToActionScheduler,
+	goToSubscriptions,
+} from '../../utils/merchant-navigation';
+
+// Run the tests if the two 'skip' environment variables are not set.
+describeif( shouldRunSubscriptionsTests && shouldRunActionSchedulerTests )(
+	'Subscriptions > Renew a subscription via Action Scheduler',
+	() => {
+		useMerchant();
+
+		const actionSchedulerHook =
+			'woocommerce_scheduled_subscription_payment';
+
+		const customerBillingConfig =
+			config.addresses[ 'subscriptions-customer' ].billing;
+
+		let subscriptionId: string;
+		let page: Page;
+
+		test.beforeAll( async ( { browser }, { project } ) => {
+			const restApi = new RestAPI( project.use.baseURL );
+			await restApi.deleteCustomerByEmailAddress(
+				customerBillingConfig.email
+			);
+
+			const { shopperPage } = await getAnonymousShopper( browser );
+			page = shopperPage;
```

Comment:
You don't need to assign `shopperPage` to `page`, to be clear in the code, it'll be probably best to use `shopperPage` instead of `page`. See my other comment about `useMerchant`.

Reply:
Okay, I was wrong, as beforeAll doesn't give you the page as `shopperPage` in this test, because it's not initialized yet. TIL that this needs to be done.

Reply:
I've fixed this up in [30107c1](https://github.com/Automattic/woocommerce-payments/pull/10143/commits/30107c107063932c0be3aaf62fbca7d5e100ffc3).